scm/github.go

```diff
@@ -709,7 +721,7 @@ func (s *GithubSCM) CreateIssue(ctx context.Context, opt *CreateIssueOptions) (*
 	if err != nil {
 		return nil, ErrFailedSCM{
 			Method:   "CreateIssue",
-			Message:  fmt.Sprintf("failed to create Issue %s, make sure all the details are correct ", opt.Title),
+			Message:  fmt.Sprintf("failed to create issue (repository %s, organization %s), make sure all the details are correct ", opt.Repository, opt.Organization),
```

Comment:
Should there be a space at the end of this string?

Also: it is not so clear what "all the details" refers to. Could this be clarified?

Reply:
PS: The log message on 719 should also be fixed... I guess this is from @oleespe's PR.